modules/hal_nordic/nrfs/backends/nrfs_backend_ipc_service.c

```diff
@@ -176,7 +176,10 @@ nrfs_err_t nrfs_backend_send(void *message, size_t size)
 
 nrfs_err_t nrfs_backend_send_ex(void *message, size_t size, k_timeout_t timeout, bool high_prio)
 {
-	if (size <= MAX_PACKET_DATA_SIZE) {
+	if (!k_is_in_isr()) {
```

Comment:
Shouldn't `&& nrfs_backend_connected()` be added here?

Reply:
added